MBBSEmu.Tests/CPU/FDIV_Tests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Iced.Intel;
+using System;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class FDIV_Tests : CpuTestBase
+    {
+        [Theory]
+        [InlineData(2, 2)]
+        [InlineData(1, .5)]
+        [InlineData(0, 0)]
```

Comment:
divide by zero is a great test, doesn't the test bomb if you attempt it?

Reply:
Nope! Even unwrapped from the unchecked it just returns `NaN`